tools/ci_build/github/azure-pipelines/qnn-ep-nuget-packaging-pipeline.yml

```diff
@@ -0,0 +1,267 @@
+      MsbuildArguments: '-detailedsummary -maxcpucount -consoleloggerparameters:PerformanceSummary'
+      OnnxRuntimeBuildDirectory: '$(Build.BinariesDirectory)'
+      DOTNET_SKIP_FIRST_TIME_EXPERIENCE: true
+      buildArch: x64
+      setVcvars: true
+      ALLOW_RELEASED_ONNX_OPSET_ONLY: '1'
+
+    steps:
+      - template: templates/set-version-number-variables-step.yml
+
+      - task: NuGetAuthenticate@0
+        inputs:
+          forceReinstallCredentialProvider: true  
+
+      - task: UsePythonVersion@0
+        inputs:
+          versionSpec: '3.8'
+          addToPath: true
+      
+      - task: PythonScript@0
+        displayName: 'Build onnxruntime for x64'
+        inputs:
+          scriptPath: '$(Build.SourcesDirectory)\tools\ci_build\build.py'
+          arguments: '--compile_no_warning_as_error --disable_ml_ops --build_dir $(Build.BinariesDirectory)\Windows --skip_submodule_sync --build_shared_lib --cmake_generator "Visual Studio 17 2022" --config ${{ parameters.build_config }} --use_qnn --qnn_home ${{parameters.qnn_sdk_path_win}}'
+
+      - template: templates/win-esrp-dll.yml
+        parameters:
+          FolderPath: '$(Build.BinariesDirectory)\Windows\${{ parameters.build_config }}\${{ parameters.build_config }}'
+          DisplayName: 'ESRP - Sign dlls'
+          DoEsrp: ${{ parameters.DoEsrp }}
+
+      - task: CmdLine@2
+        displayName: 'Generating nuspec for the native Nuget package x64'
+        inputs:
+          script: |
+            copy $(Build.SourcesDirectory)\LICENSE $(Build.SourcesDirectory)\LICENSE.txt
+            python "$(Build.SourcesDirectory)\tools\nuget\generate_nuspec_for_native_nuget.py" --package_version ${{ parameters.ort_package_version }} --package_name Microsoft.ML.OnnxRuntime.QNN --target_architecture x64 --build_config ${{ parameters.build_config }} --native_build_path=$(Build.BinariesDirectory)\Windows\${{ parameters.build_config }}\${{ parameters.build_config }} --packages_path $(Build.BinariesDirectory)\Windows\packages --ort_build_path $(Build.BinariesDirectory)\Windows --sources_path $(Build.SourcesDirectory) --commit_id $(OnnxRuntimeGitCommitHash) --is_release_build ${{ parameters.IsReleaseBuild }} --sdk_info ${{ parameters.qnn_sdk_info }}
+            cd $(Build.BinariesDirectory)\Windows\${{ parameters.build_config }}\${{ parameters.build_config }}
+            nuget pack NativeNuget.nuspec
+            mkdir $(Build.ArtifactStagingDirectory)\x64
+            copy $(Build.BinariesDirectory)\Windows\${{ parameters.build_config }}\${{ parameters.build_config }}\*.nupkg $(Build.ArtifactStagingDirectory)\x64
+
+      - task: PublishPipelineArtifact@0
+        displayName: 'Publish Pipeline x64 NuGet Artifact'
+        inputs:
+          artifactName: 'drop-winnuget-x64'
+          targetPath: '$(Build.ArtifactStagingDirectory)/x64'
+          
+  - job: OnnxRuntime_QNNNuget_Win_Arm64
+    timeoutInMinutes: 120
+
+    pool: 'onnxruntime-qnn-windows-vs-2022-arm64'
```

Comment:
If you do not the run tests, you can use a x86 machine, which in general is easier to get.